src/indev/lv_indev.c

```diff
@@ -216,7 +216,7 @@ void lv_indev_read(lv_indev_t * indev_p)
     do {
         /*Read the data*/
         indev_read_core(indev_p, &data);
-        continue_reading = data.continue_reading;
+        continue_reading = indev_p != LV_INDEV_MODE_EVENT && data.continue_reading;
```

Comment:
```suggestion
        continue_reading = indev_p->type != LV_INDEV_MODE_EVENT && data.continue_reading;
```

Reply:
Sorry, I forgot the member when replacing the function with member access. Such a simple mistake.

Thank you for reminding me. we should use the member of `mode`, and I have already updated it. Please check it.